test/src/main/java/org/apache/accumulo/test/ScanServerMetadataEntriesIT.java

```diff
@@ -258,41 +244,43 @@ public void testGcRunScanServerReferences() throws Exception {
           String file = row.substring(ScanServerFileReferenceSection.getRowPrefix().length());
           metadataScanFileRefs.add(file);
         });
-        assertEquals(3, metadataScanFileRefs.size());
+        assertEquals(fileCount, metadataScanFileRefs.size());
 
-        assertEquals(3, ctx.getAmple().getScanServerFileReferences().count());
+        assertEquals(fileCount, ctx.getAmple().getScanServerFileReferences().count());
 
         List<Reference> refs = gc.getReferences().collect(Collectors.toList());
-        refs.forEach(ref -> LoggerFactory.getLogger(ScanServerMetadataEntriesIT.class)
-            .info("REF: {}", ref.getMetadataEntry()));
-        assertTrue(refs.size() > 6);
-        List<Reference> tableRefs = new ArrayList<>();
-        refs.forEach(r -> {
-          if (r.getTableId().equals(tid) && !r.isDirectory()) {
-            assertTrue(metadataScanFileRefs.contains(r.getMetadataEntry()));
-            tableRefs.add(r);
-          }
-        });
-        LoggerFactory.getLogger(ScanServerMetadataEntriesIT.class).info("{}", tableRefs);
+        refs.forEach(ref -> log.info("REF: {}", ref.getMetadataEntry()));
```

Comment:
There might be some additional room for improvements in some of these log messages to make them more descriptive too.

Reply:
I think I was just using them for debugging.

Reply:
removed in latest commit